autoload/pandoc/table.vim

```diff
@@ -0,0 +1,721 @@
+" vim:tabstop=2:shiftwidth=2:expandtab:textwidth=99
+"
+" | Easily | manageable | text  | tables | Implementation |
+" |--------|------------|-------|--------|----------------|
+"
+" Original code credit: https://github.com/vimwiki/vimwiki/
+
+let s:textwidth = &tw
+
+function! pandoc#table#Init()
+    " Table commands
+    command! -buffer -nargs=* PandocTable call pandoc#table#create(<f-args>)
+    command! -buffer PandocTableAlignQ call pandoc#table#align_or_cmd('gqq')
+    command! -buffer PandocTableAlignW call pandoc#table#align_or_cmd('gww')
+    command! -buffer PandocTableMoveColumnLeft call pandoc#table#move_column_left()
+    command! -buffer PandocTableMoveColumnRight call pandoc#table#move_column_right()
+
+    " Table mappings
+    inoremap <expr> <buffer> <Tab> pandoc#table#kbd_tab()
```

Comment:
As @Konfekt noted in issue comments, can you please wrap these bindings in a check to make sure they are not already bound to a different key _and_ that this key isn't already bound to something else?

Reply:
This [vim-script idiom](https://github.com/Konfekt/vim-smartbraces/blob/400723ea4e60665fac417be716b0189f77d78f3f/plugin/smartbraces.vim#L43) comes to mind.